_format/1.1/index.md

```diff
@@ -49,8 +49,7 @@ The `profile` parameter is used to support [profiles].
 
 ### <a href="#content-negotiation-clients" id="content-negotiation-clients" class="headerlink"></a> Client Responsibilities
 
-Clients that include the JSON:API media type in their `Accept` header **MUST**
-specify the media type there at least once without any media type parameters.
+Clients that include the JSON:API media type in their `Accept` header **MAY** include it more than once, but for at least one instance of it, they **MUST** specify it without any media type parameters.
```

Comment:
@TrevorHinesley, thanks for doing this!

1. I think we need to add new lines after 80 characters :)
2. Do you think this is any clearer? I worked myself into a knot and trying different wordings and maybe this suggestion below is actually worse:

```
Clients that include the JSON:API media type in their `Accept` header **MAY**
include it more than once, however, at least one instance of the JSON:API media
type **MUST NOT** specify any media type parameters.
```

Reply:
Just a drive-by opinion: I think @TrevorHinesley's version reads a bit better.

Reply:
How about a slight variation of my original:

```
Clients that include the JSON:API media type in their `Accept` header **MAY** include it more than once, but at least one instance of it **MUST** be specified without any media type parameters.
```